lisa/analysis/load_tracking.py

```diff
@@ -20,6 +20,7 @@
 import pandas as pd
 
 from lisa.analysis.base import TraceAnalysisBase
+from lisa.trace import requires_one_event_of
 
 
 class LoadTrackingAnalysis(TraceAnalysisBase):
```

Comment:
I can't tag those lines on the review, but the users of `df_cpus_signals` and `df_tasks_signals` should also get the appropriate decorators.